web/src/beta/features/PluginPlayground/Plugins/presets/index.ts

```
@@ -20,6 +20,8 @@ import { featureStyle3dModel } from "./manageLayerStyle/featureStyle3dmodel";
 import { featureStyle3dTiles } from "./manageLayerStyle/featureStyle3dTiles";
 import { overrideStyle } from "./manageLayerStyle/overrideStyle";
 import { styleWithCondition } from "./manageLayerStyle/styleWithCondition";
+import { zoomInOut } from "./camera/zoomInOut";
+import { cameraRotation } from "./camera/cameraRotation";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'cameraRotation'. Consider to delete it or rename it.

'cameraRotation' is defined here:


(lint/suspicious/noRedeclare)